contentful/export-processor/content-types/section.js

```diff
@@ -152,6 +152,150 @@ export class Section {
     return minimumPathsForRecommendations;
   }
 
+  /**
+   * Calcutes paths to select each answer to each question
+   */
+    getPathsForAllAnswers() {
```

Comment:
IMO this should be split up into smaller functions. It's ~80 lines of code, and does quite a lot of complex things.

I'd take 15-20 minutes and write down, at a high level, what this method is doing. Then what is it doing step-by-step.

Then see if any of those steps are repeated. If they are, create functions that do that duplicated work, then refactor the code to use those new functions.

Once that's done, extract each big step of the function into a separate function. Refactor the function to use those new functions in place of the old code.
Then look at those smaller functions; do any of these have multiple responsibilities? If so, consider repeating the last 2 steps again.
Repeat until you're left with functions no bigger than 20 lines of code. Ideally smaller. 

Try to have every function being a functional function. Which... sounds dumb writing it like that. Let me know on Slack if you have questions around what I mean there. Or any of this.

Once all of _that_ is done, and to half answer your question about the UserJourney, consider whether any of this functionality can be moved to other classes (including new classes). What responsibilities is this `Section` class now responsible for? What are the "parent categories" of those responsibilities? How might we move those elsewhere, so we have smaller, cleaner, single responsibility, OOP-good-classes?

Reply:
I want to look at this separately, maybe in the 'break up tests' ticket but maybe as its own. For now it's working and I want to get back into dev before it gets even further away